OracleDatabase/RAC/OracleRealApplicationClusters/DOCKER_README.md

```diff
@@ -0,0 +1,669 @@
+
+## Section 5: Connecting to Oracle RAC Database
+
+**IMPORTANT:** This section assumes that you have successfully created an Oracle RAC environment.
+
+If you are using connection manager and exposed port 1521 on the host, connect from an external client using the following connection string:
+
+```
+system/<password>@//<docker_host>:1521/<ORACLE_SID>
+```
+
+If you are using the Docker MACVLAN driver and you have configured DNS appropriately, you can connect using the public scan listener directly from any external client using the following connection string:
+
+```
+system/<password>@//<scan_name>:1521/<ORACLE_SID>
+```
+
+## Section 6: Environment Variables for the first node
+
+**IMPORTANT:** This section provides details about the environment variables that can be used when creating the first node of a cluster.
+
+Parameters:
+
+```
+OP_TYPE=###Specify the Operation TYPE. It can accept 2 values INSTALL OR ADDNODE####
+
+NODE_VIP=####Specify the Node VIP###
+
+VIP_HOSTNAME=###Specify the VIP hostname###
+
+PRIV_IP=###Specify the Private IP###
+
+PRIV_HOSTNAME=###Specify the Private Hostname###
+
+PUBLIC_IP=###Specify the public IP###
+
+PUBLIC_HOSTNAME=###Specify the public hostname###
+
+SCAN_NAME=###Specify the scan name###
+
+ASM_DEVICE_LIST=###Specify the ASM Disk lists.
+
+SCAN_IP=###Specify this if you do not have DNS server###
+
+DOMAIN=###Default value set to example.com###
+
+PASSWORD=###OS password will be generated by openssl###
+
+CLUSTER_NAME=###Default value set to racnode-c####
+
+ORACLE_SID=###Default value set to ORCLCDB###
+
+ORACLE_PDB=###Default value set to ORCLPDB###
+
+ORACLE_PWD=###Default value set to generated by openssl random password###
+
+ORACLE_CHARACTERSET=###Default value set AL32UTF8###
+
+DEFAULT_GATEWAY=###Default gateway. You need this env variable if containers
+will be running on multiple hosts.####
+
+CMAN_HOSTNAME=###Connection Manager Host Name###
+
+CMAN_IP=###Connection manager Host IP###
+
+ASM_DISCOVERY_DIR=####ASM disk location insdie the container. By default it is /dev######
+
+COMMON_OS_PWD_FILE=###Pass the file name to setup grid and oracle user password. If you specify ORACLE_PWD_FILE, GRID_PWD_FILE and DB_PWD_FILE then you do not need to specify this env variable###
+
+ORACLE_PWD_FILE=###Pass the file name to set the password for oracle user.###
+
+GRID_PWD_FILE=###Pass the file name to set the password for grid user.###
+
+DB_PWD_FILE=###Pass the file name to set the password for DB user i.e. sys.###
+
+REMOVE_OS_PWD_FILES=###Set this env variable to true to remove pwd key file and password file after resetting the password.###
+
+CONTAINER_DB_FLAG=###Default value is set to true to create container database. Set this to false if you do not want to create a container database.###
+```
```

Comment:
````suggestion
```bash
OP_TYPE="[INSTALL|ADDNODE]"   # Specify the Operation TYPE. Valid values are INSTALL OR ADDNODE
NODE_VIP="1.2.3.4"  # Specify the Node VIP
VIP_HOSTNAME="vip.host.name"  # Specify the VIP hostname
PRIV_IP="1.2.3.4"  # Specify the Private IP
PRIV_HOSTNAME="priv.host.name"  # Specify the Private Hostname
PUBLIC_IP="1.2.3.4"  # Specify the public IP
PUBLIC_HOSTNAME="pub.host.name"  # Specify the public hostname
SCAN_NAME="scan.name"  # Specify the scan name
ASM_DEVICE_LIST=""  # Specify the ASM Disk lists.
SCAN_IP=""  # Specify this if you do not have DNS server
DOMAIN="example.com"  # Default value set to example.com
PASSWORD=""  # OS password will be generated by openssl
CLUSTER_NAME=""  # Default value set to racnode-c
ORACLE_SID=""  # Default value set to ORCLCDB
ORACLE_PDB=""  # Default value set to ORCLPDB
ORACLE_PWD=""  # Default value set to generated by openssl random password
ORACLE_CHARACTERSET="AL32UTF8"  # Default value
DEFAULT_GATEWAY=""  # Default gateway. You need this env variable if containers will be running on multiple hosts.
CMAN_HOSTNAME=""  # Connection Manager Host Name
CMAN_IP=""  # Connection manager Host IP
ASM_DISCOVERY_DIR=""  # ASM disk location insdie the container. By default it is /dev
COMMON_OS_PWD_FILE=""  # Pass the file name to setup grid and oracle user password. If you specify ORACLE_PWD_FILE, GRID_PWD_FILE and DB_PWD_FILE then you do not need to specify this env variable
ORACLE_PWD_FILE=""  # Pass the file name to set the password for oracle user.
GRID_PWD_FILE=""  # Pass the file name to set the password for grid user.
DB_PWD_FILE=""  # Pass the file name to set the password for DB user i.e. sys.
REMOVE_OS_PWD_FILES="[true|false]"  # Set this env variable to true to remove pwd key file and password file after resetting the password.
CONTAINER_DB_FLAG="[true|false]"  # Default value is set to true to create container database. Set this to false if you do not want to create a container database.
```
````

Or replace my suggestion with the default values for each variable.